peer.go

```diff
@@ -1245,10 +1249,12 @@ func (p *peer) handlePongMsg(msg *btcwire.MsgPong) {
 	// infrequently enough that if they overlap we would have timed out
 	// the peer.
 	if p.protocolVersion > btcwire.BIP0031Version &&
```

Comment:
I personally think a simple check for `p.lastPingNonce != 0` would be better than the protocol version check here.  The pong message didn't exist until after that protocol version and thus this function would never even be invoked on older protocol versions anyways.